free-programming-books.md

```diff
@@ -873,9 +874,9 @@
 * [Dissecting a C# Application](https://damieng.com/blog/2007/11/08/dissecting-a-c-application-inside-sharpdevelop) - Christian Holm, Bernhard Spuida, Mike Kruger
 * [Fundamentals of Computer Programming with C#](http://www.introprogramming.info/english-intro-csharp-book/read-online/) - Svetlin Nakov
 * [Introduction to Rx](http://www.introtorx.com)
+* [The SOLID Principles](http://www.c-sharpcorner.com/ebooks/the-solid-principles) - Gaurav Kumar Arora (C# Corner)
 * [Threading in C#](http://www.albahari.com/threading/)
 
-
```

Comment:
Please pay attention to the linter on Travis CI. it wants 2 blank lines after every list.